tests/Command/Issue/IssueTakeCommandTest.php

```diff
@@ -54,33 +55,16 @@ public function takes_an_issue_with_specific_base()
 
         $tester = $this->getCommandTester($command = new IssueTakeCommand());
         $command->getHelperSet()->set($this->expectTextHelper());
-        $command->getHelperSet()->set($this->expectGitHelper('origin', 'origin/development'));
+        $command->getHelperSet()->set($this->expectGitConfigHelper());
+        $command->getHelperSet()->set($this->expectGitHelper('gushphp', 'gushphp/development'));
 
         $tester->execute(
-            ['--org' => 'gushphp', 'issue_number' => TestAdapter::ISSUE_NUMBER, 'base_branch' => 'development'],
-            ['interactive' => false]
-        );
-
-        $this->assertEquals(
-            sprintf('[OK] Issue https://github.com/gushphp/gush/issues/%s taken!', TestAdapter::ISSUE_NUMBER),
-            trim($tester->getDisplay(true))
-        );
-    }
-
-    /**
-     * @test
-     */
-    public function takes_an_issue_with_specific_remote()
-    {
-        $this->expectsConfig();
-        $this->config->get('base')->willReturn('master');
-
-        $tester = $this->getCommandTester($command = new IssueTakeCommand());
-        $command->getHelperSet()->set($this->expectTextHelper());
-        $command->getHelperSet()->set($this->expectGitHelper('gushphp', 'gushphp/master'));
-
-        $tester->execute(
-            ['--org' => 'gushphp', 'issue_number' => TestAdapter::ISSUE_NUMBER, '--remote' => 'gushphp'],
+            [
+                '--org' => 'gushphp',
+                '--repo' => 'gush',
+                'issue_number' => TestAdapter::ISSUE_NUMBER,
+                'base_branch' => 'development'
```

Comment:
Ah good point, this will also break when using a completely separate issue-tracker.

We can change remote to `--source-org` and `--source-repo` to still have the automatic remote registering but allow to pull from a different org and repo. When no `--source-org` is given the `--org` is used, same goes for the repo.

Then in your local gush you can configure the default org and repo for issues (to-do).